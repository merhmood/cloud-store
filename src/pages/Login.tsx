/**
 * node_modules packages should always be addded to the section
 * denoted as node modules, while other imports should be added to
 * the section denoted as app created, this helps in organization
 * of import statements <3
 */


//app created
import Logo from '../components/Logo'
import BaseForm from '../components/BaseForm'

const Login = () => {
  return (
    <form>
      <Logo />
      <BaseForm buttonText='Log In'/>
    </form>
  )
}

export default Login