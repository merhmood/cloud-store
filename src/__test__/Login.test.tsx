/**
 * node_modules packages should always be addded to the section
 * denoted as node modules, while other imports should be added to
 * the section denoted as app created, this helps in organization
 * of import statements <3
 */

//node_modules
import React from 'react';
import renderer from 'react-test-renderer'

//app created
import Login from '../pages/Login';

it('renders Login', ()=>{
    const tree = renderer
        .create(<Login />)
        .toJSON()
    expect(tree).toMatchSnapshot()
})