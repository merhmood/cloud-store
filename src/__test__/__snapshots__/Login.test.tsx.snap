// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login 1`] = `
<form>
  <p
    style={
      Object {
        "color": "black",
        "fontSize": "2rem",
        "fontWeight": "600",
        "marginBottom": "1em",
        "textAlign": "center",
      }
    }
  >
    Media
    <span
      style={
        Object {
          "color": "#dd2222",
        }
      }
    >
      Store
    </span>
  </p>
  <input
    placeholder="Email"
    type="text"
  />
  <input
    placeholder="Password"
    type="password"
  />
  <label
    htmlFor="show password"
  >
    Show password
  </label>
  <input
    type="checkbox"
  />
  <button>
    Log In
  </button>
</form>
`;
